types/react-bootstrap-table-next/index.d.ts

```diff
@@ -108,6 +108,8 @@ export interface ColumnDescription<T extends object = any, E = any> {
      * Column header field
      */
     text: string;
+    classes?: string | (cell: T[keyof T], row: T, rowIndex: number, colIndex: number) => string;
```

Comment:
You should wrap the function in round braces, otherwise the function is not recognised as a single type.
```suggestion
    classes?: string | ((cell: T[keyof T], row: T, rowIndex: number, colIndex: number) => string);
```